brigade/templates/free_software.html

```diff
@@ -7,9 +7,9 @@
     {% if self.software() %}
       <a href="{{ url_for('.free_software_index') }}">&larr; Back to Free Software</a>
 
-      <div class="free-software__logo">
```

Comment:
I'm dropping the 'free-software' classes for now, until we work on a more 'Atomic Design' template to use for the Free Software pages.